project/src/main/webapp/event-details.js

```diff
@@ -0,0 +1,82 @@
+const eventHost = 'test@example.com'; // hard-coded event host value
```

Comment:
This will be where the other servlet (that I'll write) compares the event host from the event obj / db right?

Reply:
@yicyao I do not think you need to write a servlet you will just have to update eventHost when you fetch the event details.

Reply:
oh yes ty!